README.md

```diff
@@ -1,35 +1,71 @@
-# aerogear-otp-ios [![Build Status](https://travis-ci.org/aerogear/aerogear-otp-ios.png)](https://travis-ci.org/aerogear/aerogear-otp-ios)
+# aerogear-ios-otp
 
-## One Time Password API
+![Maintenance](https://img.shields.io/maintenance/yes/2017.svg)
+[![circle-ci](https://img.shields.io/circleci/project/github/aerogear/aerogear-ios-otp/master.svg)](https://circleci.com/gh/aerogear/aerogear-ios-otp)
+[![License](https://img.shields.io/badge/-Apache%202.0-blue.svg)](https://opensource.org/s/Apache-2.0)
+[![GitHub release](https://img.shields.io/github/release/aerogear/aerogear-ios-otp.svg)](https://github.com/aerogear/aerogear-ios-otp/releases)
+[![CocoaPods](https://img.shields.io/cocoapods/v/AeroGearOTP.svg)](https://cocoapods.org/pods/AeroGearOTP)
+[![Platform](https://img.shields.io/cocoapods/p/AeroGearOTP.svg)](https://cocoapods.org/pods/AeroGearOTP)
 
-A Objective-C iOS library for generating one time passwords according to [RFC 6238](http://tools.ietf.org/html/rfc6238).
+One time passwords generating library according to [RFC 6238](http://tools.ietf.org/html/rfc6238).
```

Comment:
As you changed the order of the words, I think here `generator` suits better than `generating`, and probably `password` better than `passwords`

Reply:
Done!